retroarch.c

```diff
@@ -1398,7 +1399,8 @@ bool retroarch_main_init(int argc, char *argv[])
    command_event(CMD_EVENT_MAPPER_INIT, NULL);
    command_event(CMD_EVENT_REWIND_INIT, NULL);
    command_event(CMD_EVENT_CONTROLLERS_INIT, NULL);
-   command_event(CMD_EVENT_RECORD_INIT, NULL);
+   if (!string_is_empty(global->record.path))
```

Comment:
This makes cli recording only start if record.path is defined.
That old codepath still works.